resources/themes/cura-light/images/first_run_Ultimaker_cloud.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="200px" height="135px" viewBox="0 0 200 135" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 52.6 (67491) - http://www.bohemiancoding.com/sketch -->
+    <title>Group-cloud</title>
+    <desc>Created with Sketch.</desc>
+    <g id="Iteration-4" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="9-Account-and-finish" transform="translate(-540.000000, -255.000000)">
+            <g id="Group-cloud" transform="translate(540.000000, 255.000000)">
```

Comment:
Unnecessary groups. You can combine the attributes.

Reply:
I'm going to leave the SVGs as they are for now.

Reply:
Removed those 2 groups in 418daae58